packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -39,43 +43,36 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
     throw new Error("Missing Courier Provider");
   }
 
-  const courierActions = useActions();
-  const [visible, setVisible] = useState(false);
-
-  const handleBellOnClick = () => {
-    setVisible(!visible);
-  };
+  const inbox = useInbox();
 
   let tippyProps: TippyProps = {
-    placement: "right",
+    trigger: props.trigger ?? "click",
+    placement: props.placement ?? "right",
     interactive: true,
   };
 
-  if (visible) {
-    tippyProps = {
-      ...tippyProps,
-      visible,
-    };
-  }
-
   useEffect(() => {
-    if (!courierActions) {
-      return;
-    }
+    courierContext.registerReducer("inbox", reducer);
```

Comment:
As much as I wish there was another way to do this. I actually like this pattern 👍🏾 
It's explicit and its abstracted.
Although I am wondering if there any race conditions with the `inbox.init`. I see the 
```
if (!courierContext?.reducers?.inbox) {
```
below

Reply:
there actually might be another way... ive been thinking about it and might play with it today